docs/Home/supported-data-serialization-formats.md

```diff
@@ -0,0 +1,13 @@
+# Data Serialization Formats
+
+MCAP files can store multiple streams of heterogeneous data, each serialized in a different data format.
```

Comment:
"Supported serialization formats for MCAP channels are listed below" - this avoids introducing a new term "streams". In mcap those are channels.